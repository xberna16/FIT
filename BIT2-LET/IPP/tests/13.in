<?xml version="1.0" encoding="UTF-8"?>
<program language="IPPcode18"><instruction order="1" opcode="DEFVAR"><arg1 type="var">GF@A</arg1></instruction><instruction order="2" opcode="DEFVAR"><arg1 type="var">GF@B</arg1></instruction><instruction order="3" opcode="DEFVAR"><arg1 type="var">GF@Vysledek</arg1></instruction><instruction order="4" opcode="MOVE"><arg1 type="var">GF@A</arg1><arg2 type="string">AAA</arg2></instruction><instruction order="5" opcode="MOVE"><arg1 type="var">GF@B</arg1><arg2 type="string">AAA</arg2></instruction><instruction order="6" opcode="WRITE"><arg1 type="string">A=AAA_B=AAA</arg1></instruction><instruction order="7" opcode="LT"><arg1 type="var">GF@Vysledek</arg1><arg2 type="var">GF@A</arg2><arg3 type="var">GF@B</arg3></instruction><instruction order="8" opcode="WRITE"><arg1 type="string">LT</arg1></instruction><instruction order="9" opcode="WRITE"><arg1 type="var">GF@Vysledek</arg1></instruction><instruction order="10" opcode="GT"><arg1 type="var">GF@Vysledek</arg1><arg2 type="var">GF@A</arg2><arg3 type="var">GF@B</arg3></instruction><instruction order="11" opcode="WRITE"><arg1 type="string">GT</arg1></instruction><instruction order="12" opcode="WRITE"><arg1 type="var">GF@Vysledek</arg1></instruction><instruction order="13" opcode="EQ"><arg1 type="var">GF@Vysledek</arg1><arg2 type="var">GF@A</arg2><arg3 type="var">GF@B</arg3></instruction><instruction order="14" opcode="WRITE"><arg1 type="string">EQ</arg1></instruction><instruction order="15" opcode="WRITE"><arg1 type="var">GF@Vysledek</arg1></instruction></program>
