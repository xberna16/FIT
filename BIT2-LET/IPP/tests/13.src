.IPPcode18
DEFVAR GF@A
DEFVAR GF@B
DEFVAR GF@Vysledek
MOVE GF@A string@AAA
MOVE GF@B string@AAA
WRITE string@A=AAA_B=AAA
LT GF@Vysledek GF@A GF@B
WRITE string@LT
WRITE GF@Vysledek
GT GF@Vysledek GF@A GF@B
WRITE string@GT
WRITE GF@Vysledek
EQ GF@Vysledek GF@A GF@B
WRITE string@EQ
WRITE GF@Vysledek
