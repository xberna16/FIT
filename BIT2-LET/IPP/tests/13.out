A=AAA_B=AAA
LT
false
GT
false
EQ
true
